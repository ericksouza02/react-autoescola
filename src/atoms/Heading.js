import React from "react";
import styled from "styled-components";
import {breakAt, BreakPointSizes} from '../styles/BreakPointSize';
import ThemeProvider from "../styles/ThemeProvider";

const Heading = styled.h1`
    position: relative;
    margin-bottom: 25px;
    padding-bottom: 25px;
    border-bottom: 1px solid rgba(255, 255, 255, 0.2);

    &::after {
        content: "";
        position: absolute;
        left: 0;
        bottom: -3px;
        background-color: ${props => props.theme.colors.primary.main};
        height: 5px;
        width: 70px;
    }

    h1, 
    h2, 
    h3, 
    h4, 
    h5,
    h6 {
       margin-bottom: 0;
    }
`

export default Heading;
