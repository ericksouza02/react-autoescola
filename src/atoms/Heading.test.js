import React from "react";
import { render } from "@testing-library/react"
import Heading from "./Heading";

test('match snapshot component Heading', () => {

    const {asFragment} = render(<Heading/>)

    expect(asFragment).toMatchSnapshot()
})