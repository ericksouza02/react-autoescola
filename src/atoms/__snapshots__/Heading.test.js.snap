// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`match snapshot component Heading 1`] = `[Function]`;
